src/coreclr/jit/emitarm64.cpp

```diff
@@ -11869,10 +11910,12 @@ void emitter::emitIns_R_R_R_I(instruction ins,
 
         default:
             // fallback to emit SVE instructions.
-            return emitInsSve_R_R_R_I(ins, attr, reg1, reg2, reg3, imm, opt, attrReg2);
+            return emitInsSve_R_R_R_I(ins, attr, reg1, reg2, reg3, imm, opt, sopt);
```

Comment:
I wonder if we should just call `emitInsSve*` directly from the unit tests? That way, we don't pollute the non-SVE paths with SVE-specific logic. (We can also come back later and split out the implementations during cleanup.)